src/content/protocols/ethena.md

```diff
@@ -0,0 +1,158 @@
+---
+protocol: "Ethena"
+website: "https://ethena.fi/"
+x: "https://x.com/ethena_labs"
+github: "https://github.com/ethena-labs"
+defillama_slug: ["ethena"]
+chain: "Ethereum"
+stage: 0
+risks: ["L", "L", "H", "H", "M"]
+author: ["andrijdavid"]
+submission_date: "2024-12-10"
+publish_date: "1970-01-01"
+acknowledge_date: "1970-01-01"
+update_date: "1970-01-01"
+---
+
+# Summary
+
+
+Ethena is a synthetic dollar protocol built on Ethereum that provides a crypto-native solution for money not reliant on traditional banking system infrastructure. It introduces the 'Internet Bond,' a globally accessible dollar-denominated rewards instrument. Ethena's synthetic dollar, USDe, is fully backed by staked crypto assets (Etherum and Bitcoin) and its corresponding short futures positions. Ethena employs delta-hedging strategies to maintain stability and generate yields for users.
+
+# Overview
+
+## Chain
+
+The Ethena protocol is deployed on Ethereum mainnet.
+
+> Chain score: L
+
+## Upgradeability
+
+Ethena's smart contracts are not upgradable but the ability to set and update roles introduces a level of flexibility and risk. 
+
+> Upgradeability: L
```

Comment:
Hey @andrijdavid 

Maybe this was not clear enough from our framework description, but in the upgradability section & risk we address bytecode upgrades and parameter changes that are permissioned.

And as you rightfully discovered there are many permissioned functions, so each function with a permission needs to be considered when determining the risk on `Upgradability`

To iterate on the assessment
Does calling one of the permissioned functions result
- in loss/blocking of user funds
- loss of unclaimed yield
- change expected behavior significantly (blacklisting/kyc/fees/...)

Please review the permissions and assess the impact and come to a conclusion on the upgradability risk.

Some help when writing the impact column below, think of least 2,3 sentences
First sentence: what it does technically, e.g "It assigns a new address to the owner variable"
Second: what is the impact within the system, e.g "The owner is permissioned to raise fees"
Third: Imagine faulty or malicious action, e.g "The malicious owner could raise fees to 100%, redirecting all future yield.

Reply:
also this contract is missing in the analysis https://etherscan.io/address/0x7fd57b46ae1a7b14f6940508381877ee03e1018b#code